modules/connectors/ninja_van/karrio/mappers/ninja_van/proxy.py

```diff
@@ -0,0 +1,53 @@
+"""Karrio Ninja Van client proxy."""
+
+import karrio.lib as lib
+import karrio.api.proxy as proxy
+import karrio.mappers.ninja_van.settings as provider_settings
+
+
+class Proxy(proxy.Proxy):
+    settings: provider_settings.Settings
+
+    def get_rates(self, request: lib.Serializable) -> lib.Deserializable[str]:
+        response = lib.request(
+            url=f"{self.settings.server_url}/service",
+            data=lib.to_json(request.serialize()),
+            trace=self.trace_as("json"),
+            method="POST",
+            headers={},
+        )
+
+        return lib.Deserializable(response, lib.to_dict)
+
+    def create_shipment(self, request: lib.Serializable) -> lib.Deserializable[str]:
+        response = lib.request(
+            url=f"{self.settings.server_url}/service",
```

Comment:
Similarly, this one  needs to be updated to reflect the [Order API](https://api-docs.ninjavan.co/en#tag/Order-API/paths/~1%7BcountryCode%7D~14.2~1orders/post)

```suggestion
            url=f"{self.settings.server_url}/4.2/orders",
```